Brickwall/models.py

```diff
@@ -0,0 +1,67 @@
+from sqlalchemy import Column, Integer, String, DateTime, Boolean, ForeignKey
+from . import db
+from enum import Enum
+
+
+
+class Review(db.Model):
+    __tablename__ = "review"
+    review_id = Column(Integer, primary_key=True)
+    # user name of reviewer
+    member = Column(String)
+    # whether the person worked full time or co-op
+    # takes a string, but precondition is it should take one of two strings ("full-time", "co-op")
+    type = Column(String)
+    company_id = Column(ForeignKey('company.company_id'))
+
+class Location:
+    __tablename__ = "location"
+    location_id = Column(Integer, primary_key=True)
+    # name of the location
+    name = Column(String)
+    # map_location - TBD
+
+class Interview:
+    __tablename__ = "interview"
+    interview_id = Column(Integer, primary_key=True)
+    review_id = Column(ForeignKey('review.review_id')) 
+    # number of times the person interviewed
+    interview_count = Column(Integer, default=0)
```

Comment:
Move the comments into the schema itself. Then it will show in sql also
```suggestion
    interview_count = Column(Integer, default=0, comment='number of times the person interviewed')
```